Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -7,6 +7,9 @@
 
 @interface NSManagedObject (VOKManagedObjectAdditions)
 
+/// A completion block after an asynchronous operation on a temporary context. NSManagedObject's are not threadsafe.
+typedef void(^VOKManagedObjectReturnBlock)(NSArray *arrayOfManagedObjects);
```

Comment:
Since this gets an array, should it be called `VOKManagedObjectsReturnBlock` (plural objects)?